scripts/sender.py

```diff
@@ -1,12 +1,24 @@
 #!/usr/bin/env python
 __author__ = 'flier'
+
+import argparse
+
 import rospy
 import leap_interface
 from leap_motion.msg import leap
 from leap_motion.msg import leapros
 
+FREQUENCY_ROSTOPIC_DEFAULT = 0.01
+PARAMNAME_FREQ = '/leapmotion/freq'
```

Comment:
I would not recommend defining an absolute parameter name.

A better solution is to use a private parameter: ``~freq``. In the usual case, that resolves to ``/leapmotion/freq``, but it also works correctly when run in a different namespace or with a different node name.